lerna.json

```diff
@@ -0,0 +1,7 @@
+{
+  "lerna": "2.7.1",
```

Comment:
i strongly suggest using independent versioning if you're publishing multiple packages. Causing version churn unnecessarily is a large burden on downstream consumers.

Reply:
At this moment, I wanted to use `lerna` only for `bootstrap` command. I absolutely agree that the independent versioning is better, but in some cases, it could be problematic.

Reply:
I'm not familiar with any that are problematic; certainly if you're just using bootstrap, that's fine :-) that's what enzyme is doing (but we still turned on independent versioning just in case) https://github.com/airbnb/enzyme/blob/master/lerna.json#L6-L7

Reply:
Ok, I'll add it just in case, thanks -)